test/test_toroid.py

```diff
@@ -0,0 +1,183 @@
+# A script to run test cases of toroid-ray.py
+
+
+# TODO: do this in a more pythonic way
```

Comment:
Do you find anything on this?

Reply:
Oh yeah, that's the way to do it. Then put the directory containing the `__init__.py` on you `$PYTHONPATH`